lib/uplink-gomobile/release/release.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -ueo pipefail
+set +x
+
+if [ $# -eq 0 ]
+then
+      echo "Usage: release.sh version"
+      exit 1
+fi
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# setup tmpdir for testfiles and cleanup
+TMP=$(mktemp -d -t tmp.XXXXXXXXXX)
+cleanup(){
+	rm -rf "$TMP"
+}
+trap cleanup EXIT
+
+VERSION=$1
+
+make libuplink-gomobile
+
+# create versioned copies of files
+cp -v "$SCRIPTDIR/libuplink-android-gomobile.pom"   "$TMP/libuplink-android-gomobile-$VERSION.pom"
+cp -v "libuplink-android.aar"                       "$TMP/libuplink-android-gomobile-$VERSION.aar"
+cp -v "libuplink-android-sources.jar"               "$TMP/libuplink-android-gomobile-sources-$VERSION.jar"
+cp -v "$SCRIPTDIR/AndroidManifest.xml"              "$TMP/AndroidManifest.xml"
+
+# set version for pom file
+sed -i 's@<version></version>@<version>'$VERSION'</version>@g' "$TMP/libuplink-android-gomobile-$VERSION.pom"
+# set version for AndroidManifest.xml
+sed -i 's@android:versionName=""@android:versionName="'$VERSION'"@g' "$TMP/AndroidManifest.xml"
+# set versionCode for AndroidManifest.xml
+CODE=$(date +%s) # maybe there is better way to generte versionCode
```

Comment:
Typo in "generte". And, yes, let's think if there is a better way for autoincrementing code.

Reply:
for now, replaced autogenerated `versionCode` with parameter like `version`